hermes/crates/cardano-chain-follower/Cargo.toml

```diff
@@ -9,3 +9,8 @@ repository.workspace = true
 license.workspace = true
 
 [dependencies]
+pallas.workspace = true
```

Comment:
As above, you should not be modifying anything to do with cardano-chain-follower in this PR.